site/src/docs/components/header/index.mdx

```diff
@@ -17,61 +17,289 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
+        { label: 'English', value: 'en' },
+      ]} />
+      <Header.ActionBarItem fullWidth label="Haku" icon={IconSearch} id="action-bar-search">
+        <Header.NavigationSearch label="Hae palvelusta" onChange={() => {}} onSubmit={() => {}} />
+      </Header.ActionBarItem>
+    </Header.ActionBar>
+    <Header.NavigationMenu>
+      <Header.NavigationLink
+        href="#"
+        label="Sosiaali- ja terveyspalvelut"
+        onClick={(event) => event.preventDefault()}
+        active
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Terveydenhoito"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Hammashoito" />,
+              <Header.NavigationLink href="#" label="Julkinen terveydenhoito" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Senioripalvelut"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Viriketoiminta" />,
+              <Header.NavigationLink href="#" label="Kuntouttavat palvelut" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink
+        href="#"
+        label="Kasvatus ja koulutus"
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Kasvatus"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Varhaiskasvatus" />,
+              <Header.NavigationLink href="#" label="Esiopetus" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Koulutus"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Perusopetus" />,
+              <Header.NavigationLink href="#" label="Toisen asteen koulutus" />,
+              <Header.NavigationLink href="#" label="Työväenopistot" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink href="#" label="Asuminen" />
+    </Header.NavigationMenu>
   </Header>
 </PlaygroundPreview>
 
 ### Principles
 
-- **It is strongly recommended to always include the header component in your service.**
+- **It is strongly recommended to always include the header component on every page in your service.**
 - The Header component is a wrapper for the HDS header navigation components. This means you should put all the other header elements inside this component.
 - The only mandatory component inside the header is HeaderActionBar. Other components may be used if needed.
-- HeaderNavigationBar is coming soon
-- HeaderActionBar is coming soon
 - The inner components should **always** be in this order: Header.SkipLink, Header.UniversalBar, HeaderActionBar and HeaderNavigationBar.
+- Try to keep the order of actions (language select, search, log in) consistent in navigation. Do not change the default order without a good reason.
 
```

Comment:
To better understand how navigation elements work and how they should be combined, see also Navigation pattern (link)